student/urls.py

```diff
@@ -22,6 +22,7 @@
   url(r'^user/logout/$', 'django.contrib.auth.views.logout', {'next_page': '/'}),
   url(r'^unsubscribe/(?P<id>[\w.@+-]+)/(?P<token>[\w.:\-_=]+)/$', student.views.unsubscribe),
   url(r'^user/settings/$', student.views.UserView.as_view()),
+  url(r'^user/settings/delete/$', student.views.UserView.as_view()),
```

Comment:
why add the separate endpoint here? I think it's more RESTful to just user `user/settings`